### PR TITLE
chore: July 2020 windows patches

### PR DESCRIPTION
**Pull request by AbelHu**
**Reason for Change**:

Adds release notes and updates default windows image to use July's patches

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -188,8 +188,9 @@ function Install-WindowsPatches
     # Windows Server 2019 update history can be found at https://support.microsoft.com/en-us/help/4464619
     # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
-    # KB4551853 contains May 2020 cumulative updates for Windows Server 2019
-    $patchUrls = @("http://download.windowsupdate.com/c/msdownload/update/software/secu/2020/06/windows10.0-kb4561608-x64_31903b9b843877474d625e2b3beb5fed3b8abc3a.msu")
+    # KB4558998 contains July 2020 cumulative updates for Windows Server 2019
+    # https://www.catalog.update.microsoft.com/Search.aspx?q=4558998
+    $patchUrls = @("http://download.windowsupdate.com/c/msdownload/update/software/secu/2020/07/windows10.0-kb4558998-x64_6da68fe659dacb747458ab3a431c3546ce7765b5.msu")
```
Comment (jsturtevant) [inline vhd/packer/configure-windows-vhd.ps1:193]: It looks like the market place image is avaliable and the image version below contains this patch?  If so we dont have to apply the patch and can leave this blank.  

This is for cases where the latest image isn't available and we have to use last month's image

Comment (AbelHu) [reply vhd/packer/configure-windows-vhd.ps1:193]: I just followed the guidance to check the market place image. I should not update `vhd/packer/windows-vhd-builder.json`

**Comment by AbelHu** [comment]
> Can you also add the hotfix packages and latest k8s patch release?

A separate PR is created. https://github.com/Azure/aks-engine/pull/3599

**Comment by jsturtevant** [comment]
I believe the code coverage change is because a couple comment lines where added.  The versions are covered in VHD pipeline tests

**Pull request merged**